### DISCUSSION TITLE
Modulo operator in slint script

### DISCUSSION THREAD

**Discussion by loloof64** [slint-ui/slint#4190] [Q&A] [answered]
Hi ! Is there a modulo operator in *.slint syntax ?

So that something like 10 % 9 returns 1 (the remainder of the division).

I've been looking searching in the documentation, but I did not manage to find if it is defined somewhere.

**Comment by loloof64** [comment, marked as answer]
Sorry, I've just found from the Intellisense in my Vs Code editor, that there's a "mod(value, divisor)" function already.

**Reply by ogoffart** [reply]
You're not the first having trouble to find it.
I've just added an error message that should help the next one running into this: https://github.com/slint-ui/slint/pull/4191
